templates/page.tpl.php

```diff
@@ -89,16 +89,22 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
     <div class="wrapper">
       <section class="footer-top">
         <nav>
-          <?php print render($page['footer']); ?>
+          <?php print render($page['footer_top']); ?>
         </nav>
       </section>
+      <section>
+        <div class="footer-logo"></div>
+        <div class="footer-links">
+          <nav>
+            <?php print render($page['footer']); ?>
+          </nav>
```

Comment:
Yeah we may need to break footer into 3 regions:

footer-top
footer-navigation
footer (bottom)